test/e2e/helloworld_shell_test.go

```diff
@@ -53,6 +54,15 @@ func cleanup(yamlFilename string, logger *logging.BaseLogger) {
 	os.Remove(yamlFilename)
 }
 
+func serviceHostname() string {
+	return noStderrShell("kubectl", "get", "rt", "route-example", "-o", "jsonpath={.status.domain}", "-n", test.ServingNamespace)
+}
+
+func ingressAddress(gateway string, addressType string) string {
+	return noStderrShell("kubectl", "get", "svc", gateway, "-n", "istio-system",
+		"-o", fmt.Sprintf("jsonpath={.status.loadBalancer.ingress[*]['%v']}", addressType))
```

Comment:
`%v` → `%s`?

Reply:
I think `%v` is good enough. A snippet from https://golang.org/pkg/fmt/:
`%v`	 the value in a default format
`%s`	 the uninterpreted bytes of the string or slice
Is there a reason to change it?

Reply:
I am proponent of stronger typing, since it expresses the intent more clearly.
In addition, would your change still work if you change`'%v'` → `%q` (note the lack of single quotes).